cps/src/test/groovy/org/jenkinsci/plugins/workflow/cps/LogActionTest.groovy

```diff
@@ -51,6 +51,6 @@ echo("Hello I'm Gilbert")
         assert exec.isComplete()
         FlowNode atom = exec.currentHeads[0].parents[0]
         LogActionImpl la = atom.getAction(LogAction)
-        assert la.logFile.text=="Hello I'm Gilbert\n"
+        assert la.logFile.text.trim() == "Hello I'm Gilbert"
```

Comment:
this brings an intresting question - what will happen if you have both an echo in a windows node and an echo in a Linux node?
WIll the console render correctly - will it contain spruious \r if the main task in on a linux host  - or will lines be munged if this is on a Windows host?